Xamarin.Essentials/Screenshot/MediaFile.shared.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Xamarin.Essentials
+{
+    public class ScreenshotFile : ReadOnlyFile, IDisposable
+    {
+        public bool Disposed { get; private set; }
+
+        internal ScreenshotFile(string filePath)
+            : base(filePath, "image/png")
+        {
+        }
+
+#if !NETSTANDARD1_0
+        public Stream AsStream() => System.IO.File.Open(FullPath, FileMode.Open, FileAccess.Read);
```

Comment:
I would suggest making this `Task<Stream> PlatformOpenReadStreamAsync()` so that it matches the FilePicker: https://github.com/xamarin/Essentials/pull/975/files#diff-afb1518dfb7ed9f416a31b1bd082b4d6R113

I am hoping to have all the pickers inherit from `PickerResultBase` (or a better named class) so that the `ReadOnlyFile` is more for the sending out and we have a "picker" type file that has the extras for opening it and maybe a method (like you have here on the dispose) to clean up.

Reply:
Also, with the #if, make sure you add a dummy method for netstandard1.0 so that the API exists for the compiler.

```csharp
#if NETSTD1
  Task<Stream> PlatformOpenReadStreamAsync() => Task.FromResult<Stream>(null);
#else
  Task<Stream> PlatformOpenReadStreamAsync() => ...;
#endif
```

Reply:
Sure can do, how would you feel about making the actual filestream returned async as well? We currently create the stream via File.Open which does not allow us to specify the async'ness of the resulting stream - if we call the filestream constructor directly we will be able to specify it to be an async stream.